apps/mediator/src/index.ts

```diff
@@ -13,13 +13,29 @@ void createAgent().then(async (agent) => {
     role: OutOfBandRole.Sender,
```

Comment:
if `recreateInvitation` is set to true, we can probably skip this whole query and immediately create the invitation. If you create a new invitation on every startup, this query will return more and more results every time it starts up.

I'd write it like this:
- if recreate invitation
	- create new invitation and log it
- else fetch all existing invitation
  - if existing invitation log it
  - else create invitation and log it